kubernetes/elastic-helm/deployment.yaml

```diff
@@ -90,6 +90,8 @@ opentelemetry-collector:
       otlp/elastic:
         endpoint: ${env:ELASTIC_APM_ENDPOINT}
         compression: none
+        tls:
+          insecure: true
```

Comment:
Hmmm, can we make it an env var? `${eq (lower ${env:ELASTIC_APM_INSECURE_TLS} "true"}` . I _think_ that should do it

Reply:
Hm could be, I have no idea how to debug whether that results in the right value so I will remove for now and we can revisit in follow up. I've Googled about this a bit and I cannot figure out why/how we are using this `${env:` syntax in helm, no docs seem to reference it!

Reply:
We talked, I've put that back in but without any parameterization for now. It's a demo so we think it will be safe. I've added a comment and we can circle back later. I unfortunately do not know how to test whether the interpolation of the env var produces the correct result.